gubled/benchmarking_test.go

```diff
@@ -107,6 +114,8 @@ func TestThroughput(t *testing.T) {
 	totalMessages := testgroupCount * messagesPerGroup
 	throughput := float64(totalMessages) / end.Sub(start).Seconds()
 	log.Printf("finished! Throughput: %v/sec (%v message in %v)", int(throughput), totalMessages, end.Sub(start))
+
+	time.Sleep(time.Second * 5)
```

Comment:
when the test stops the temporary dir is deleted leaving the fetch process trying to get some messages from some deleted files.